modules/apps/frontend-taglib/frontend-taglib-clay/.npmbundlerrc

```diff
@@ -112,6 +112,5 @@
 		"xss-filters": true
 	},
 	"output": "build/node/packageRunBuild/resources",
-	"preset": "liferay-npm-bundler-preset-standard",
```

Comment:
I'm not super confident about this change... this can have different effects. We should be able to simply add the `scss` loader rule or others so that our build diff is minimal.

Reply:
How to do this? I've seen the option to extend preset [here](https://github.com/liferay/liferay-frontend-projects/tree/master/projects/npm-tools/packages/npm-scripts#preset), is that the way to add loader rule? What is the syntax for it? This should be a fairly common use, it's odd that there is no example, making me doubt I am looking in the right place. I also tried looking in all configuration in portal and couldn't find anything sass-related, except for few [webpack configs](https://github.com/liferay/liferay-portal/blob/master/modules/dxp/apps/commerce/commerce-bom-web/webpack.config.dev.js#L50-L55) that seem unrelated.

Reply:
I answered this [in Slack](https://liferay.slack.com/archives/G3JBR21HA/p1604993425437400?thread_ts=1604921984.431300&cid=G3JBR21HA); you need [config like this](https://github.com/liferay/liferay-frontend-projects/blob/8c6cc33e718ac2f2bcd233d7310a72ffa2cd4785/projects/npm-tools/packages/npm-bundler-preset-liferay-dev/config.json#L270-L287).

Reply:
Got it, thanks!